packages/web-client/src/helpers/space/functions.ts

```diff
@@ -60,24 +78,33 @@ export function buildPublicSpaceResource(data): PublicSpaceResource {
 }
 
 export function buildShareSpaceResource({
+  driveAliasPrefix,
   shareId,
   shareName,
   serverUrl
 }: {
+  driveAliasPrefix: 'share' | 'ocm-share'
   shareId: string | number
   shareName: string
   serverUrl: string
 }): ShareSpaceResource {
+  let id
+  if (driveAliasPrefix === 'ocm-share') {
+    id = `${OCM_PROVIDER_ID}$${shareId}!${shareId}`
+  } else {
+    id = [SHARE_JAIL_ID, shareId].join('!')
+  }
```

Comment:
Afaict this is the only place where we need to distinguish between regular `share` and `ocm-share`  in the drive alias -  do we want to introduce this new drive alias prefix just for this?

An alternative would be a query param or find another clever solution to delay using the id until the actual share is loaded...